spec/taskBehaviors.js

```diff
@@ -0,0 +1,267 @@
+describe('Tasks', function() {
+    beforeEach(function() {
+        jasmine.Clock.useMockForTasks();
+    });
+
+    afterEach(function() {
+        // Check that task schedule is clear after each test
+        expect(ko.tasks.length()).toEqual(0);
+    });
+
+    it('Should run in next execution cycle', function() {
+        var runCount = 0;
+        ko.tasks.schedule(function() {
+            runCount++;
+        });
+        expect(runCount).toEqual(0);
+
+        jasmine.Clock.tick(1);
+        expect(runCount).toEqual(1);
+    });
+
+    it('Should run multiple times if added more than once', function() {
+        var runCount = 0;
+        var func = function() {
+            runCount++;
+        };
+        ko.tasks.schedule(func);
+        ko.tasks.schedule(func);
+        expect(runCount).toEqual(0);
+
+        jasmine.Clock.tick(1);
+        expect(runCount).toEqual(2);
+    });
+
+    it('Should run scheduled tasks in the order they were scheduled', function() {
+        var runValues = [];
+        var func = function(value) {
+            runValues.push(value);
+        };
+
+        ko.tasks.schedule(func.bind(null, 1));
+        ko.tasks.schedule(func.bind(null, 2));
+
+        jasmine.Clock.tick(1);
+        expect(runValues).toEqual([1,2]);
+    });
+
+    it('Should run tasks again if scheduled after a previous run', function() {
```

Comment:
Again, I had copied this from my plugin, which has the ability to provide arguments separately. The argument isn't necessary for the test, so I'll change it to use the same function instance.